### ISSUE TITLE
Implement ApplicationLogJSONEnabled for ApplicationLog json formatted log lines

### ISSUE THREAD

**Issue by tachang** [zalando/skipper#1494]
**Is your feature request related to a problem? Please describe.**
We log everything in JSON and for some logs such as the ApplicationLog it does not seem to result in JSON.
I am not sure if this is a configuration error or not but I don't see it.

**Describe the solution you would like**
A setting called ApplicationLogJSONEnabled to output JSON

**Describe alternatives you've considered (optional)**
Basically pipelines or filtering to turn it into JSON (outside of skipper).

**Comment by szuecs** [comment]
@tachang sounds like a good enhancement. Do you want to create a PR, adding flag and config entry for having application logs also switchable to JSON?

**Comment by szuecs** [comment]
@tachang do you have opinions on https://github.com/zalando/skipper/pull/1496#discussion_r465616065 ?
If not we would merge this as it is.
